html/partials/compare/changed_config_files.html.haml

```diff
@@ -23,10 +23,14 @@
             - if @diff["changed_config_files"].only_in1
               = only_in_a
               = render_partial "compare/changed_config_files_list", list: @diff["changed_config_files"].only_in1
+            - else
+              No individual changed config files found.
```

Comment:
For the message maybe we could use a helper method so in case we do a change we don't need to change it on every instance. We could pass the scope name as an argument.

Also not sure if the word "individual" is necessary here.